technical-reports/format/composite-types.md

```diff
@@ -423,7 +423,7 @@ Describes a gradient that is solid yellow for the first 2/3 and then fades to re
 {
   "brand-primary": {
     "$type": "color",
-    "$value": "#99ff66"
+    "$value": "#00ff66"
```

Comment:
```suggestion
    "$value": {
      "colorSpace": "srgb",
      "channels": [0, 1, 0.4]
```

Reply:
OK! Yes breaking out changes to gradient as a separate PR would be easier to review! This PR shouldn’t introduce any changes to gradient. Because, yes, gradients do use colors, that doesn’t mean gradients change necessarily. Then we can review those unique changes easier